src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -447,16 +449,14 @@ export default {
 @import '../../../../../assets/variables';
 
 .message-main {
-	display: flex;
+	display: grid;
+	grid-template-columns: minmax(0, $messages-text-max-width) 1fr;
 	justify-content: space-between;
```

Comment:
It's not consistent for all messages
![image](https://github.com/user-attachments/assets/ab29d8fa-e6e4-4161-ae1e-48ac247d11b5)

Reply:
adressed with 8ch for now